### PR TITLE
Added solution

### PR DESCRIPTION
**Pull request by IlyaGrynyshyn**

### CODE REVIEW

**Review by pavlejviki — CHANGES_REQUESTED** [review]
tests failed

**Comment by IlyaGrynyshyn** [comment]
Which tests are failed?

File: services/user.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from django.db.models import QuerySet
+
+from db.models import User
+
+
+def create_user(username: str,
```
Comment (Polyakiv-Andrey) [inline services/user.py:6]: After opening the parentheses, write each the arguments on a new line, including the first one

**Review by Polyakiv-Andrey — CHANGES_REQUESTED** [review]

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

**Review by viktoria-rybenchuk — APPROVED** [review]
Well done!

**Pull request closed without merging**